packages/osd-ui-framework/src/global_styling/variables/_font.scss

```diff
@@ -1,6 +1,6 @@
 // Font
 
-$kuiFontFamily: $euiFontFamily;
+$kuiFontFamily: "var(--font-text)";
```

Comment:
curious - is there really a difference in this case?

Reply:
This case and current theme structure, no. These will meet the other CSS driven styles.